app/assets/javascripts/pretty-text/addon/pretty-text.js

```diff
@@ -39,6 +39,8 @@ export function buildOptions(state) {
     customEmojiTranslation,
     watchedWordsReplace,
     watchedWordsLink,
+    featuresOverride,
```

Comment:
I would have loved to just name this `features` but it is already taken where we use it to selectively disable features. For example, you would pass in `{ features: {table: false} }` to disable the table markdown engine. In my case, I only want the features which I pass in to be enabled so I can pick and choose what I want to enable.